index.js

```diff
@@ -106,11 +109,19 @@ function qTest(runner, options = {}) {
 
       test.qTest.executionLog.status = stateFailed
       test.qTest.executionLog.note = err.stack
-      // test.qTest.executionLog.attachments: [{
-      //   name: 'screenshot.png',
-      //   content_type: 'image/png',
-      //   data: 'base64 string of Sample.docx'
-      // }]
+      
+      const attachmentsFolder = path.join(process.cwd(), `${attachmentFolder}`)
+
+      if (fs.existsSync(attachmentsFolder)) {
```

Comment:
I'm fine with the approach, however you may also take a look at https://github.com/adamgruber/mochawesome#addcontexttestobj-context

It looks a bit more flexible imo. 
We can have global after each hook, in case test failed add attachment to test object and then extract it here.

That's up to you